python/lsst/ts/ataos/ataos_csc.py

```diff
@@ -107,6 +111,12 @@ def __init__(self, config_dir=None, initial_state=State.STANDBY):
                                               "reportedFilterPosition",
                                               "reportedDisperserPosition"])
 
+        # self.atcs = ATCS(intended_usage=7)  # this is OffsettingForATAOS Usage
+
+        self.log.debug('loading ATCS from ataos_csc')
```

Comment:
Not sure I get the need for this debug message here... :/

But if you think it is useful, try to format a bit better.

```
        self.log.debug('Loading ATCS from ATAOS.')
```

maybe?

Reply:
removed.